src/commons/components/ceph-dashboard/ceph-dashboard.scss

```diff
@@ -0,0 +1,80 @@
+.tendrl-ceph-dashboard-view-container {
+
+	background-color: $color-light-grey;
```

Comment:
@gnehapk which indentation is not correct can you be specific? Can you please point out the specifics.